### PR TITLE
Explorer UI Update: Added all relevant token logos and symbols for CCTP addition

### PR DESCRIPTION
**Pull request by Defi-Moses**
**Description**
There were lots of missing token Logos and symbols due to LinkedPool integrations, CCTP on new chains etc. This added all of those to the Explorer UI

**Additional context**
Add any other context about the problem you're solving.

**Metadata**
- Fixes #[Link to Issue]

### Summary by CodeRabbit

- New Feature: Enhanced the display of token symbols in the Explorer UI. If a token symbol is unavailable, a placeholder ("--") will be shown, improving the user experience.
- New Feature: Expanded token support across various chains. New tokens including CRVUSDC, USDBC, USDTE, and BTCB have been added, and existing tokens have been updated with additional addresses and decimals for different chains.
- New Feature: Added support for the UNIDX token on multiple chains including BASE, OPTIMISM, and FANTOM.
- New Feature: Introduced the AVAX token for the KLAYTN chain, broadening our token offerings.
- Style: Cleaned up code by removing unused import statements and trailing commas.

226fcfc17f85f192c6ef35d3412322552f996c05: [explorer-ui preview link ](https://sanguine-k99cptufh-synapsecns.vercel.app)
2918796cd7926aabfd3226ebc7fbfc1f1ded5878: [explorer-ui preview link ](https://sanguine-efun5qibf-synapsecns.vercel.app)

### CODE REVIEW

**Comment by ChiTimesChi** [comment]
@Defi-Moses we might also want to transform `USDC` and `CCTP_USDC` into `USDC` and `USDC.e`. Probably better in a separate PR / track as issue for now.

File: packages/explorer-ui/constants/tokens/basic.ts [hunk 1]
```diff
@@ -1081,8 +1173,14 @@ export const BASIC_TOKENS_BY_CHAIN = {
     ETH,
     NETH,
     WETH,
+    USDC,
```
Comment (ChiTimesChi) [inline packages/explorer-ui/constants/tokens/basic.ts:1176]: Probably don't need `USDC` here, as it doesn't have a `ChainId.BASE` key at this point

File: packages/explorer-ui/constants/tokens/basic.ts [hunk 2]
```diff
@@ -265,6 +285,28 @@ export const DAI = new Token({
   swapableType: 'USD',
 })
 
+export const CRVUSDC = new Token({
```
Comment (ChiTimesChi) [inline packages/explorer-ui/constants/tokens/basic.ts:288]: Small nit: this should be `CRVUSD` instead

**Review by ChiTimesChi — APPROVED** [review]
LGTM with a few minor thing outlined below

**Comment by Defi-Moses** [comment]
> @Defi-Moses we might also want to transform `USDC` and `CCTP_USDC` into `USDC` and `USDC.e`. Probably better in a separate PR / track as issue for now.

@ChiTimesChi Working on a brand new explorer that feels much more like etherscan and built off of token types in synapse-interface (and files like bridgeable.ts) to decrease complexity of making these small changes. Thats my long term solution here but probably should clean this up as well

**Pull request merged**